lua/weapons/gmod_tool/stools/starfall_screen.lua

```diff
@@ -1,4 +1,5 @@
-TOOL.Category		= "Wire - Display"
+TOOL.Category		= "Visuals/Screens"
+TOOL.Wire_MultiCategories = { "Chips, Gates" } -- Also add this to the chips, gates category so that it can be found near the processor
```

Comment:
Before this pull request, the screen and processor were in completely different categories. After this pull request, they are still in completely different categories (visuals/screens vs chips, gates). These two categories are where all the wire tools that used to be in `Wire - Display` and `Wire - Control` were moved to. I simply wanted it to be exactly the same as before.

However, after I'd done that, I thought, hey since wire now supports tools being in multiple categories at once, why not also add the screen to the same categories as the processor? It makes sense to have them near each other, but it also makes sense for one to be in the display category and the other in the gates, chips category, so why not both?